app/views/home/_form.html.erb

```diff
@@ -1,36 +1,29 @@
-  <div class="modal-dialog modal-dialog-centered">
-    <div class="modal-content">
-      <div class="modal-header bg-gradient-primary-to-secondary p-4">
-        <h5 class="modal-title font-alt text-white" id="feedbackModalLabel">Contact Us</h5>
-        <button class="btn-close btn-close-white" type="button" data-bs-dismiss="modal" aria-label="Close"></button>
-      </div>
-      <div class="modal-body border-0 p-4">
-        <%= form_with model: @order, local: true do |form| %>
-          <div class="form-floating mb-4">
-            <!--            <input class="form-control" id="inputName" type="text" placeholder="Enter your name..." />-->
-            <%= form.text_field :name, class: "form-control", placeholder:"Enter your name...", id:"inputName"%>
-            <label for="inputName", class="labelinput"><%= form.label :name%></label>
-          </div>
-          <div class="form-floating mb-4">
-            <!--            <input class="form-control" id="inputEmail" type="email" placeholder="name@example.com" />-->
-            <%= form.text_field :email, class: "form-control", placeholder:"name@example.com", id:"inputEmail"%>
-            <label for="inputEmail", class="labelinput"><%= form.label :email %></label>
-          </div>
-          <div class="form-floating mb-4">
-            <!--            <input class="form-control" id="inputPhone" type="tel" placeholder="(123) 456-7890" />-->
-            <%= form.text_field :phone, class: "form-control", placeholder:"(123) 456-7890", id:"inputPhone"%>
-            <label for="inputPhone", class="labelinput"><%= form.label :phone %></label>
-          </div>
-          <div class="form-floating mb-4">
-            <!--            <textarea class="form-control" id="inputMessage" placeholder="Enter your message here..." style="height: 10rem"></textarea>-->
-            <%= form.text_field :comments, class: "form-control", placeholder:"Enter your message here...", id:"inputMessage"%>
-            <label for="inputMessage", class="labelinput"><%= form.label :comments %></label>
-          </div>
-          <div class="d-grid"><%= form.submit class:"btn btn-primary rounded-pill py-3" %></div>
-        <% end %>
-      </div>
+<div class="modal-header bg-gradient-primary-to-secondary p-4">
+  <h5 class="modal-title font-alt text-white" id="feedbackModalLabel">Contact Us</h5>
+  <button class="btn-close btn-close-white" type="button" data-bs-dismiss="modal" aria-label="Close"></button>
+</div>
+<div class="modal-body border-0 p-4">
+  <%= form_with model: @order, local:false do |form| %>
+    <div class="form-floating mb-4">
+        <%= form.text_field :name, class: @order.errors[:name].any? ? 'form-control input-error' : 'form-control', placeholder:"Enter your name...", id:"inputName" %>
+        <label for="inputName", class="<%=@order.errors[:name].any? ? 'error_explanation' : 'labelinput' %>"><%= form.label :name %></label>
```

Comment:
any specific reason to wrap form.label in <label>? couldn't we just use form.label?